rdfs-improved/README.md

```diff
@@ -1234,3 +1199,139 @@ Here are the current results, but it should be rerun after fixes to ontology: se
 | xsd:string    |  51 |                                                                                     |
 
 I have a tentative SPARQL Update, but need to revise it.
+
+# Fix Technical Notes
```

Comment:
@Sveino @griddigit-ci This is the basis of reproducible SPARQL fixes: the naming, sequence (so they can be concatenated).
Please read and give feedback